stdlib/event/gui.lua

```diff
@@ -99,6 +99,9 @@ end
 -- @return (<span class="types">@{Gui}</span>)
 function Gui.remove(event_ids, gui_element_pattern, handler)
     fail_if_missing(event_ids, "missing event argument")
+    if not (Is.Nil(gui_element_pattern) or Is.String(gui_element_pattern)) then
```

Comment:
It is always mostly my fault
```lua
fail_if_missing(Is.Nil(gui_element_pattern) or Is.String(gui_element_pattern), guil_element_pattern argument must be a string when present)
````
This will have consistent stack trace
Ps think of fail_if_missing as just fail_if_falsy(condition, message),  I may change it at some point

Reply:
Understood, sorted downstream.